AsyncDisplayKit/Private/ASImageNode+AnimatedImagePrivate.h

```diff
@@ -0,0 +1,27 @@
+//
+//  ASImageNode+AnimatedImagePrivate.h
```

Comment:
I would actually really prefer if we named this or put it in a +Internal or +FrameworkPrivate, some generally-named thing as a core class of this sort is pretty likely to have additions (and creating files for each is suboptimal for organizing the structure of the project)